docs/index.md

```diff
@@ -0,0 +1,78 @@
+# Getting Started with TwinTrim
+
+Welcome to the TwinTrim getting started guide! This document will walk you through the steps to get started with the project.
+
+### Prerequisites
+
+Before you begin, ensure you have met the following requirements:
+
+- **Python Installation**: You have installed the latest version of Python.
+- **GitHub Account**: You have a GitHub account.
+
+### Installing TwinTrim
+
+To install TwinTrim, follow these steps:
+
+1. **Clone the Repository**: Clone the repository from GitHub.
+
+    ```bash
+    git clone https://github.com/Kota-Karthik/twinTrim
+    ```
+
+2. **Navigate to Project Directory**: Navigate to the project directory.
+
+    ```bash
+    cd twinTrim
+    ```
+
+3. **Install Dependencies**: Install the required dependencies using the following command:
+
+   ```bash
+   pip install -r requirements.txt
+   ```
+
+### Running TwinTrim
+
+To run TwinTrim, follow these steps:
+
+1. **Navigate to Project Directory**: Navigate to the project directory.
+2. **Run the Application**: Run the following command:
+
+   ```bash
+   python app.py
+   ```
+
+The application will start running on your local server. You can access it by navigating to `http://localhost:5000` in your web browser.
```

Comment:
you do not run it on server. This is a python package which is accessed via command line.